hcsctl/e2e/ceph/ceph_test.go

```diff
@@ -0,0 +1,82 @@
+package testCeph
```

Comment:
Same as well, by convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. It's under e2e directory, so I think it's okay to have rook as a package name. Did you encounter error message for that?